src/ralph/lib/information_bubble/migrations/0003_auto_20250312_1327.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 2.2.28 on 2025-03-12 13:27
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('auth', '0011_update_proxy_permissions'),
+        ('information_bubble', '0002_servicebasedinformationbubble_groups'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='servicebasedinformationbubble',
+            name='groups',
+        ),
```

Comment:
so migration 0002 is not really needed?

Reply:
Yeah, they could've been squashed into 1. Now it's 1